planner/core/tiflash_selection_late_materialization.go

```diff
@@ -69,11 +69,13 @@ func predicatePushDownToTableScan(sctx sessionctx.Context, plan PhysicalPlan) Ph
 	case *PhysicalTableReader:
 		predicatePushDownToTableScan(sctx, p.tablePlan)
```

Comment:
Can you add tests for it?

Reply:
have added before, due to didnot merge  master.

Reply:
https://github.com/pingcap/tidb/pull/42349